clang/include/clang/Basic/TokenKinds.def

```diff
@@ -1002,6 +1002,7 @@ PRAGMA_ANNOTATION(pragma_riscv)
 ANNOTATION(module_include)
 ANNOTATION(module_begin)
 ANNOTATION(module_end)
+ANNOTATION(module_name)
```

Comment:
The place is not good. The comments above implies that this is for clang header modules while it is not the case.

Reply:
```suggestion

// Annotations for c++ named modules
ANNOTATION(module_name)
```

Reply:
Yeah, I've removed `c++` words in the comments,  because it's also used for clang modules and objective-c modules.